src/3d.js

```diff
@@ -96,15 +168,43 @@ export var hackOptions = {
 	// 	- 'box': standard cube
 	// 	- 'floor': plane flat on the ground
 	//  - 'tower1', 'tower2', etc: box variations that are taller and tiled
+    //  - 'wedgeFloor': base mesh for wedges, facing left with its slanted side
 	getType: function (drawing) {
 		var drw = drawing.drw;
+		var name = drawing.name || '';
+		// specific
```

Comment:
and then it would also need to check if there is this kind of template to prevent errors? like
`
var meshMatch = name.match(/#mesh\((.+?)\)/);
if (meshMatch) {
    if (meshTemplates[meshMatch[1]]) {
         return meshMatch[1];
     } else {
        console.log(`${meshMatch[1]} is not a valid mesh`)
     }
}
`

Reply:
it wouldn't need to, but that'd be a nice addition! (i'd use `throw new Error('<reason its an error>')` instead of just a console log though)